front_end/webserver.py

```diff
@@ -88,7 +91,7 @@ def make_app():
         url(r"/test", TestHandler, name="test"),
     ], autoescape=None)
 
-#        url(r"\/initialize", InitializeHandler, name="initialize"),
+    # url(r"\/initialize", InitializeHandler, name="initialize"),
```

Comment:
Please remove this commented line of code because I don't think we'll need it anymore.